src/dev/build/tasks/os_packages/docker_generator/templates/dockerfile.template.js

```diff
@@ -74,6 +74,9 @@ function generator({ artifactTarball, versionTag, license, usePublicArtifact  })
   # variables and translate them to Kibana CLI options.
   COPY --chown=1000:0 bin/kibana-docker /usr/local/bin/
   
+  ${ !imageFlavor ? '#Add certs for Code to clone from common sites' : '' }
```

Comment:
Can we include this in a function with a self explanatory name that returns a string as we did for `copyArtifactTarballInsideDockerOptFolder` ?

Reply:
done